src/FormatInput.js

```diff
@@ -1,3 +1,30 @@
+/*
+The file as the name suggests is used to format input as recieved from user
+and provide a more informative and documented one. Here is a example list of few
+expansions to the below used short-forms:
+
+o_string: optional string
+
+string_a: an array of strings
+
+o_string_a: an optional array of strings
+
+and similarly for others.
+
+For various format inputs, the meaning they convey are:
+
+'+' : add a step
+'-' : remove a step
+'^' : insert a step (in between)
+'r' : run a step
+'l' : load image
+
+*/
+
+
```

Comment:
I suggest removing some of these blank lines

Reply:
Yeah sure!
Thanks!!